DESCRIPTION

```diff
@@ -55,3 +55,4 @@ Imports:
     gt,
     glue,
     gtExtras
+
```

Comment:
shiny is missing here - that is where the warning comes from and why the check fails ....